rln-wasm/src/lib.rs

```diff
@@ -160,10 +159,24 @@ impl<'a> ProcessArg for &'a [u8] {
     }
 }
 
+impl ProcessArg for Endianness {
+    type ReturnType = Endianness;
+
+    fn process(self) -> Self::ReturnType {
+        self
+    }
+}
+
 #[allow(clippy::not_unsafe_ptr_arg_deref)]
 #[wasm_bindgen(js_name = newRLN)]
-pub fn wasm_new(zkey: Uint8Array) -> Result<*mut RLNWrapper, String> {
-    let instance = RLN::new_with_params(zkey.to_vec()).map_err(|err| format!("{:#?}", err))?;
+pub fn wasm_new(zkey: Uint8Array, endianness: Number) -> Result<*mut RLNWrapper, String> {
```

Comment:
Would it be more explicit to pass a boolean here and so user cannot pass any invalid value here?

Reply:
hm, and variable name will be something like (`is_little_endian`)?  Or having name like `endianness` for bool vakue is good enough?

Reply:
is_little_endian looks more explicit I would say